src/vmm/src/vmm_config/entropy.rs

```diff
@@ -0,0 +1,109 @@
+// Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+use std::ops::Deref;
+use std::sync::{Arc, Mutex};
+
+use devices::virtio::rng::{Entropy, Error as EntropyError};
+use serde::{Deserialize, Serialize};
+
+/// This struct represents the strongly typed equivalent of the json body from entropy device
+/// related requests.
+#[derive(Debug, Clone, PartialEq, Eq, Deserialize, Serialize)]
+#[serde(deny_unknown_fields)]
+pub struct EntropyDeviceConfig {}
+
+impl From<&Entropy> for EntropyDeviceConfig {
+    fn from(_dev: &Entropy) -> Self {
+        EntropyDeviceConfig {}
+    }
+}
+
+/// Errors that can occur while handling configuration for
+/// an entropy device
+#[derive(Debug, thiserror::Error)]
+pub enum EntropyDeviceError {
+    /// Error while create an entropy device
+    #[error("Could not create Entropy device: {0}")]
+    CreateDevice(#[from] EntropyError),
+}
```

Comment:
Is it expected that additional variants will be added? If not this can be more simply represented as a newtype:
```suggestion
/// Errors that can occur while handling configuration for
/// an entropy device
#[derive(Debug, thiserror::Error)]
#[error("Could not create Entropy device: {0}")]
pub struct EntropyDeviceError(#[from] EntropyError);
```

Reply:
yes, after this one gets merged I'll open another PR that will actually add more variants.